aten/src/ATen/native/quantized/cpu/qnnpack/src/fc-run.cc

```diff
@@ -98,6 +98,12 @@ enum pytorch_qnnp_status qnnpackLinear(
       .ukernel = pytorch_qnnp_params.q8conv.gemm,
   };
 
+  if (output_size == 0) {
```

Comment:
Why not use batch_size==0 to bail out in qlinear.cpp itself? To allocate zero sized tensor with appropriate dims and return that?

Reply:
I think as many functions as possible should accept bs=0.  If I put the check higher up in qlinear.cpp, then t his function needs to document that bs=0 is not supported, and someone will probably eventually forget that and call it anyway.  This fix puts the bail-out right next to where the semantically invalid operation (using 0 for tile size) occurs.

Reply:
Good point.